src/contracts/Protocol/QuestionAPI.sol

```diff
@@ -121,6 +121,10 @@ contract QuestionAPI is Ownable, NFTLocked {
         _questionStateController.setDisqualifiedState(questionId);
     }
 
+    function setOffsetForQuestionState(uint256 newOffset) public onlyOwner {
```

Comment:
would this work if there are multiple people using the api at the same time?  should the offset be a method parameter instead?